### DISCUSSION TITLE
Opening sidebar doesn't work after navigating to page

### DISCUSSION THREAD

**Discussion by DeNIKE666** [inertiajs/inertia#1134] [Help]
**Hello**, I'm new to **Inertia Js**, I'm making a user account, and I have a problem, if I refresh the page then the sidebar works, if I go to the page through <Link :href="route('dashboard')">Dashboard<Link/> then the sidebar doesn't respond

![image](https://user-images.githubusercontent.com/30469085/158978268-b28f80d1-ea50-44c8-b0dd-ef248b646ecc.png)

How can I solve this problem?

**Comment by jonagoldman** [comment]
Imposible to answer this without your source code.
